app/build.gradle

```diff
@@ -125,18 +125,18 @@ dependencies {
     // architecture components
     implementation 'androidx.lifecycle:lifecycle-extensions:2.1.0'
     //room
-    implementation 'androidx.room:room-runtime:2.1.0'
+    implementation 'androidx.room:room-runtime:2.2.0'
```

Comment:
Should we try incremental annotation processing?
https://developer.android.com/jetpack/androidx/releases/room#2.2.0

Reply:
It can make the build faster, right? I don't have any problems with the build time but yes, lets try it.